build.sbt

```diff
@@ -105,24 +110,39 @@ lazy val scafi = project.in(file("."))
     addMappingsToSiteDir(ScalaUnidoc / packageDoc / mappings, ScalaUnidoc / siteSubdirName)
   )
 
-lazy val commonsCross = crossProject(JSPlatform, JVMPlatform).in(file("commons"))
+lazy val commonsCross = crossProject(JSPlatform, JVMPlatform, NativePlatform).in(file("commons"))
   .settings(commonSettings: _*)
   .settings(
     name := "scafi-commons"
   )
   .jsSettings(
     libraryDependencies += "org.scala-js" %%% "scalajs-java-time" % "1.0.0"
   )
+  .nativeSettings(
+    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.4.0-M1",
+    nativeConfig ~= {
+      _.withLTO(LTO.thin)
```

Comment:
nativeConfiguration seems to be the same in different subprojects. @KevinManciniHull Have you seen if it is possible to avoid this repetition? (i.e. as we have done with commonSettings?